sklearn/cluster/_k_means_elkan.pyx

```diff
@@ -18,7 +18,7 @@ from ..metrics import euclidean_distances
 from ._k_means import _centers_dense
 
 
-cdef floating euclidian_dist(floating* a, floating* b, int n_features) nogil:
+cdef floating euclidean_dist(floating* a, floating* b, int n_features) nogil:
```

Comment:
@lesteve that's what I thought. should @taehoonlee then prepend an underscore to make it clear to us in the future?